pkg/workload/tpcc/timeutil.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2024 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package tpcc
+
+import (
+	"time"
+
+	"github.com/cockroachdb/cockroach/pkg/util/syncutil"
+	"github.com/cockroachdb/cockroach/pkg/util/timeutil"
+	"golang.org/x/exp/rand"
+)
+
+var TpccTime = &tpccTime{}
```

Comment:
why instantiate tpccTime at a global level instead of at the worker level?

Reply:
It felt similar to the `RandomSeed` global variable we had, which is why I went with a global declaration. Do you think it should instead be passed in as an argument?

Reply:
i think `initializeTpccTime` should return a fresh tpccTime every time it is called, otherwise, every time somebody calls `initializeeTpccTime` the global state will be clobbered.

also, this naming is more conventional in go: `newTpccTime()`